peps/pep-0751.rst

```diff
@@ -177,6 +177,14 @@ All keys listed below are required unless otherwise noted. If two keys are
 mutually exclusive to one another, then one of the keys is required while the
 other is disallowed.
 
+Key order in tables -- including the top-level table -- SHOULD be emitted by
+lockers in the order they specified in this PEP when applicable and not
+otherwise specified to minimize noise in diffs. If the keys are not explicitly
```

Comment:
This sentence doesn't make sense to me and I'm not sure what you're trying to say. I think "they" should be removed, and I'm not sure you need "and not otherwise specified".